gateway/src/node/client_handling/websocket/connection_handler/eth_events.rs

```diff
@@ -159,7 +176,7 @@ impl Detokenize for Burned {
     where
         Self: Sized,
     {
-        if tokens.len() != 3 {
+        if tokens.len() != 4 {
```

Comment:
why 4? should it be a constant instead?

Reply:
It's the number in the `Burned` struct, don't really see it necessary to be a constant tbh as it's already in the same file with the structure.